src/record_syscall.cc

```diff
@@ -1626,16 +1626,20 @@ static Switchable prepare_ioctl(RecordTask* t,
     case IOCTL_MASK_SIZE(VIDIOC_ENUM_FMT):
     case IOCTL_MASK_SIZE(VIDIOC_ENUM_FRAMESIZES):
     case IOCTL_MASK_SIZE(VIDIOC_ENUM_FRAMEINTERVALS):
+    case IOCTL_MASK_SIZE(VIDIOC_ENUMINPUT):
     case IOCTL_MASK_SIZE(VIDIOC_G_FMT):
     case IOCTL_MASK_SIZE(VIDIOC_S_FMT):
     case IOCTL_MASK_SIZE(VIDIOC_TRY_FMT):
     case IOCTL_MASK_SIZE(VIDIOC_G_PARM):
     case IOCTL_MASK_SIZE(VIDIOC_S_PARM):
     case IOCTL_MASK_SIZE(VIDIOC_REQBUFS):
     case IOCTL_MASK_SIZE(VIDIOC_QUERYBUF):
+    case IOCTL_MASK_SIZE(VIDIOC_QUERYCTRL):
     case IOCTL_MASK_SIZE(VIDIOC_QBUF):
     case IOCTL_MASK_SIZE(VIDIOC_G_CTRL):
+    case IOCTL_MASK_SIZE(VIDIOC_G_OUTPUT):
     case IOCTL_MASK_SIZE(VIDIOC_S_CTRL):
+    case IOCTL_MASK_SIZE(VIDIOC_STREAMOFF):
```

Comment:
Is this needed? I see
```
#define VIDIOC_STREAMOFF	 _IOW('V', 19, int)
```
So, consistent with documentation, `VIDIOC_STREAMOFF` doesn't change any memory and we should have returned at the end of the `if (!(_IOC_READ & dir))` block above.